python/dgl/transform.py

```diff
@@ -1295,6 +1295,13 @@ def to_simple(g, return_counts='count', writeback_mapping=False, copy_ndata=True
         from the original graph. If False, the simple
         graph will not have any node features.
         (Default: True)
+    copy_edata: bool, optional
+        If True, the edge features of the simple graph are lazy-copied
+        from the original graph. If there exists duplicate edges between
+        two vertexes (u, v), the edge feature of the simpled edge is
+        randomly selected from one of the duplicate edges.
```

Comment:
"the edge feature of the simpled edge is randomly selected from one of the duplicate edges." -> "the feature of the edge is randomly selected from one of the duplicate edges."